docassemble/AssemblyLine/data/questions/al_package.yml

```diff
@@ -1,3 +1,7 @@
+################
+# Deprecated as of 1/11/2022
+# When enough packages point to assembly_line.yml, this file should turn into an alias
+# that just includes assembly_line.yml
```

Comment:
Any reason that you made the includes chain different between AL and AL unstyled?

* `assembly_line.yml` -> `al_package.yml` -> everything
vs
* `al_package_unstyled.yml` -> `assemble_line_unstyled.yml` -> everything

The latter seems easier, and isolates the deprecated `al_package.yml` a bit more. I don't think it breaks anything either.

Reply:
My reasoning was that al_package_unstyled.yml is only referenced in a very small number of interviews, while al_package is referenced in many files. Adding the extra include adds a ms or two of load time to the interview, so I was trying to reduce impact on interviews "in the wild". But a few ms isn't a big deal. I can be persuaded to handle both the same way.

Reply:
That makes sense. You could say that the extra ms or so is encouragement to switch to the faster loading include :wink:

Reply:
Fair enough! Maybe I'll switch it to the more "correct" way now instead of waiting.